ihmc-communication/src/main/java/us/ihmc/communication/crdt/CRDTBidirectionalBoolean.java

```diff
@@ -10,13 +10,13 @@
  */
 public class CRDTBidirectionalBoolean
 {
-   private final RequestConfirmFreezable requestConfirmFreezable;
+   private final LatestTimestampModifiable latestTimestampModifiable;
```

Comment:
Switched the implementation for all behavior code from RequestConfirmFreezable to the new LatestTimestampModifiable. The scene graph still uses RequestConfirmFreezable.